src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private Random randomChoice = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private Figure[] figures = {new IsoscelesTrapezoid(getRandomNumber(),

```

Comment:
Create a private method for each figure that will create a figure with random parameters

Reply:
not fixed